### ISSUE TITLE
Cannot find module 'ember-cli-babel/package'

### ISSUE THREAD

**Issue by tomlagier** [ember-cli/ember-cli-htmlbars-inline-precompile#82]
https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/commit/4524dd130da507d07264f02bb98524573ae5acc3#diff-168726dbe96b3ce427e7fedce31bb0bcR18

That guy is causing our builds to break. Here's a truncated package.json that you should be able to reproduce with:

```
"ember-cli": "2.11.0",
"ember-cli-babel": "^5.1.7",
"ember-cli-htmlbars-inline-precompile": "^0.3.3",
```

ember-cli-babel is resolving to 5.2.4 and ember-cli-htmlbars-inline-precompile is resolving to 0.3.10

**Comment by blimmer** [comment]
👍 this just started happening to me.

**Comment by leighfarrell** [comment]
Experiencing the same issue, with slightly more outdated versions of `ember-cli` and `ember-cli-babel`:

```
"ember-cli": "2.8.0",
"ember-cli-babel": "5.1.6",
"ember-cli-htmlbars-inline-precompile": "^0.3.1"
```

`ember-cli-htmlbars-inline-precompile` is also resolving to `0.3.10` (note the caret).

**Comment by blimmer** [comment]
Locking to `"ember-cli-htmlbars-inline-precompile": "0.3.9"` is a quick workaround until a fix is published.

**Comment by rwjblue** [comment]
😭 so sorry y'all 😢 

Released 0.3.11, please test and let me know 😥...

**Comment by tomlagier** [comment]
Working for me. Thanks for the quick turnaround @rwjblue , your hard work is very much appreciated :)
